retriever/lib/get_opts.py

```diff
@@ -52,9 +49,6 @@
 default_parser = subparsers.add_parser('defaults', help='displays default options')
 update_parser = subparsers.add_parser('update', help='download updated versions of scripts')
 new_parser = subparsers.add_parser('new', help='create a new sample retriever script')
```

Comment:
Is it possible to make this new parser create scripts within the scripts folder

Reply:
We can so it, should I open a separate pr for it or do it in this?

Reply:
Let's open a separate PR for this